mappings/net/minecraft/world/chunk/light/ChunkLightProvider.mapping

```diff
@@ -15,6 +15,7 @@ CLASS net/minecraft/class_3558 net/minecraft/world/chunk/light/ChunkLightProvide
 		ARG 1 array
 		ARG 2 blockPos
 	METHOD method_15518 hasUpdates ()Z
+	METHOD method_15520 getDebugLightValue (J)Ljava/lang/String;
```

Comment:
`getLightDebugString` (or just `getDebugString`, since this is a light provider class).